packages/ag-charts-community/src/scene/shape/line.ts

```diff
@@ -42,8 +42,9 @@ export class Line extends Shape {
         return new BBox(this.x1, this.y1, this.x2 - this.x1, this.y2 - this.y1);
     }
 
-    isPointInPath(_x: number, _y: number): boolean {
-        return false;
+    isPointInPath(px: number, py: number): boolean {
+        const { x, y } = this.transformPoint(px, py);
+        return this.computeBBox().grow(this.strokeWidth).containsPoint(x, y);
```

Comment:
As discussed offline: This is incorrect for non-vertical/non-horizontal lines (diagonal line being the most pessimistic case) - but I think it's an ok compromise unless there is a straightforward solution.

We might be able to use something related to https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line#Line_defined_by_two_points (although I don't think this will tell you if the distance calculated is to a point past either end of the line)

Reply:
oh, didn't think about that..
I initially tried this solution, but it didn't work because I didn't know yet about `transformPoint`..
I can give it another try, but I'm wondering, would it be more efficient to use the existing logic for straight lines?
we could do a simple check if the x's or y's are identical

Reply:
LGTM now - thanks!